packages/core/src/app/checkout/Checkout.spec.tsx

```diff
@@ -273,7 +273,7 @@ describe('Checkout', () => {
     });
 
     it('calls trackStepViewed when a step is expanded', async () => {
-        jest.useFakeTimers();
+        jest.useFakeTimers({ legacyFakeTimers: true });
```

Comment:
Need to add this config after upgrading jest `28.0.0`
https://jest-archive-august-2023.netlify.app/docs/28.x/upgrading-to-jest28#jestusefaketimers